graphene_sqlalchemy/tests/test_types.py

```diff
@@ -224,6 +224,19 @@ class Meta:
     assert pets_field.type().description == 'Overridden'
 
 
+def test_invalid_model_attr():
```

Comment:
FYI I had to add an unrelated test because `coveralls` thought incorrectly that the overall coverage went down without this.